packages/expo/bundledNativeModules.json

```
@@ -110,5 +110,6 @@
   "unimodules-image-loader-interface": "~6.1.0",
   "@shopify/react-native-skia": "v2.0.0-next.2",
   "@shopify/flash-list": "1.7.6",
-  "@sentry/react-native": "~6.10.0"
+  "@sentry/react-native": "~6.10.0",
+  "react-native-bootsplash": "~6.3.7"
```

```suggestion
  "react-native-bootsplash": "^6.3.7"
```

we could also use ^ to be less restrictive, in case new minor bumps are released